.eslintrc.js

```diff
@@ -17,6 +17,8 @@ module.exports = {
     // TODO: fix copied js code that makes this warn necessary
     'no-use-before-define': ['warn'],
     'linebreak-style': 0,
+    //TODO: make requests asynchronous
+    'no-await-in-loop': ['warn'],
```

Comment:
This is a bad sign. It usually implies that you can do
```js
await Promise.all(
  list.map(
    item => expensiveAsyncOps(item)
  )
);
```
instead of
```js
for (const item of list) {
  await expensiveAsyncOps();
}
```